editoast/simulation/src/envs/core.rs

```diff
@@ -0,0 +1,19 @@
+use std::sync::Arc;
+
+/// Innermost simulation environment to configure Core basic parameters
+pub struct CoreEnv {
+    pub infra_id: u64,
+    pub infra_version: i64,
+    pub client: Arc<core_client::CoreClient>,
+}
```

Comment:
Do you plan to add the electrical profile ID for the simulation?

Reply:
In the `SimulationEnv` directly as it is irrelevant for pathfinding.